src/bundle/Controller/SearchController.php

```diff
@@ -110,7 +110,9 @@ public function searchAction(Request $request): Response
         $creator = null;
         $contentTypes = [];
         $lastModified = $search['last_modified'] ?? [];
+        $lastModifiedSelect = $search['last_modified_select'] ?? '';
```

Comment:
What does it mean "lastModifiedSelect" and what's the different between $lastModified? It's not readable now, please use more descriptive name if you really need(?) add these properties.
Analogue to created and createdSelect.